addon/utils/get-with-default.js

```diff
@@ -0,0 +1,11 @@
+import { get } from '@ember/object';
+
+export default function getWithDefault(objectInstance, key, defaultValue) {
+  let value = get(objectInstance, key);
+
+  if (value === undefined || value === null) {
```

Comment:
I agree that a `null` check is clearer, but I feel like this could technically be a breaking change. If a user explicitly has `flash. destroyOnClick` set as `null`, Ember right now returns null.  This change would give them a default value instead.

Can we have it just mimic the current `getWithDefault` implementation?

Reply:
> this could technically be a breaking change

Yeah, I've been thinking about this for the past few days. To be safe, this would have to be a major version bump, so perhaps it's just easier to stick with the same behaviour. I'll push an update.